quality/pmd/android.gradle

```diff
@@ -8,7 +8,7 @@ pmd {
   ruleSetFiles = files(rootProject.file('config/quality/pmd/pmd-ruleset.xml'))
   ruleSets = [] // see https://stackoverflow.com/a/34145485
   targetJdk = TargetJdk.VERSION_1_7
-  toolVersion = '5.8.1' // see https://github.com/pmd/pmd/releases
+  toolVersion = '6.0.0' // see https://github.com/pmd/pmd/releases
```

Comment:
why not '6.34.0' ? Is there a way to specify version "up to next major"?

Reply:
no particular reason.
updated to latest.